docs/operation/training.rst

```diff
@@ -107,15 +288,14 @@ Build and Install ACT Models
     ├── LICENSE
     └── README.md
 
-
-Navigate to the ``detr`` directory inside the repository and install the detr module whihc contains the model definitions using the below command:
+Navigate to the ``act_plus_plus`` directory inside the repository and install the ``detr`` module, which contains the model definitions, by running the following command:
```

Comment:
Is this really required? Do we need to mention detr?

Reply:
No we don't. This one looks better.